x/oracle/README.md

```diff
@@ -58,6 +58,8 @@ Validators must first pre-commit to a set of exchange rates, then in the subsequ
 
   Voters that have managed to vote within a narrow band around the median are rewarded with a portion of the collected seigniorage.
 
+  The set of validators that can earn rewards is updated every `SlashWindow` by taking all the active validators at the end of a `SlashWindow` and updating the `ValidatorRewardSet`. Any validators that join in the middle of a `SlashWindow` will not be able to earn ballot rewards until the next `SlashWindow`.
```

Comment:
Add a note here on how we treat validators which leave in the middle of the reward window, and what we do with the rewards that validators are ineligible for.

Do we send them to the community pool? Just ignore them?

Reply:
Also technically this happens at the start of every `SlashWindow` right?